packages/client/src/v2-events/components/forms/inputs/FileInput/SimpleDocumentUploader.tsx

```diff
@@ -0,0 +1,175 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import React, { useState } from 'react'
+import {
+  injectIntl,
+  WrappedComponentProps as IntlShapeProps,
+  MessageDescriptor
+} from 'react-intl'
+import styled from 'styled-components'
+import { FieldValue, FileFieldValue } from '@opencrvs/commons/client'
+import { ErrorText } from '@opencrvs/components/lib/ErrorText'
+import { ImageUploader } from '@opencrvs/components/lib/ImageUploader'
+import { buttonMessages, formMessages as messages } from '@client/i18n/messages'
+import { DocumentPreview } from './DocumentPreview'
+import { DocumentListPreview } from './DocumentListPreview'
+
+const DocumentUploader = styled(ImageUploader)`
+  color: ${({ theme }) => theme.colors.primary};
+  background: ${({ theme }) => theme.colors.white};
+  border: ${({ theme }) => `2px solid ${theme.colors.primary}`};
+  border-radius: 4px;
+  ${({ theme }) => theme.fonts.bold14};
+  height: 40px;
+  text-transform: initial;
+
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.md}px) {
+    margin-left: 0px;
+    margin-top: 10px;
+  }
+`
+
+const FieldDescription = styled.div`
+  margin-top: 0px;
+  margin-bottom: 6px;
+`
+
+type IFullProps = {
+  name: string
+  label: string
+  file?: FileFieldValue
+  description?: string
+  allowedDocType?: string[]
+  error?: string
+  disableDeleteInPreview?: boolean
+  onComplete: (file: File | null) => void
+  touched?: boolean
+  onUploadingStateChanged?: (isUploading: boolean) => void
+  requiredErrorMessage?: MessageDescriptor
+  previewTransformer?: (files: FileFieldValue) => FileFieldValue
+} & IntlShapeProps
+
+function SimpleDocumentUploaderComponent({
+  allowedDocType,
+  name,
+  onUploadingStateChanged,
+  intl,
+  previewTransformer,
+  onComplete,
+  label,
+  file,
+  description,
+  error: errorProps,
+  disableDeleteInPreview,
+  requiredErrorMessage,
+  touched
+}: IFullProps) {
+  const [error, setError] = useState('')
```

Comment:
We could just use undefined value

Reply:
Yea, this was some og code brought back from the core side. Don't want to touch it quite yet